spec/models/pageflow/widget_spec.rb

```diff
@@ -6,18 +6,6 @@ module Pageflow
       Pageflow.config.widget_types.clear
     end
 
-    describe '.copy_all_to' do
-      it 'copies all widgets to given subject' do
-        theming = create(:theming)
-        revision = create(:revision)
-        create(:widget, subject: theming, role: 'header', type_name: 'custom_header')
-
-        theming.widgets.copy_all_to(revision)
-
-        expect(revision.widgets).to include_record_with(type_name: 'custom_header', role: 'header')
-      end
-    end
-
```

Comment:
This has been [added back in](https://github.com/scrollytelling/pageflow/commit/2bc90b326d24058e8ad74f9d6c708f3f25f3bdf3).